src/app/lists/CommentBox.tsx

```diff
@@ -73,10 +74,22 @@ const SuccessText = styled.p`
     font-size: 0.8rem;
 `;
 
+const WarningText = styled.p`
+    color: ${(props) => props.theme.primary.background[3]};
+    margin: 1rem 0 0;
+    font-size: 0.8rem;
+`;
+
 const CommentBox: React.FC<CommentProps> = ({ originalComment }) => {
     const [value, setValue] = useState(originalComment);
     const [errorMessage, setErrorMessage] = useState("");
     const [successMessage, setSuccessMessage] = useState("");
+    const [unsavedChanges, setUnsavedChanges] = useState(false);
```

Comment:
the variable name `unsavedChanges` would imply a list of unsaved changes. We could rename it to something like `hasUnsavedChanges`?

I also don't think we actually need a state here. Since `hasUnsavedChanges` is just depends on `value` and `originalComment`, we could do
```ts
const hasUnsavedChanges = value !== originalComment
```
and this would get updated whenever one of them changes.